dist/utils/validate-ast.spec.js

```diff
@@ -0,0 +1,113 @@
+"use strict";
```

Comment:
So far I haven't been able to get mocha to transpile the tests. Working on this now.

Reply:
Did you try `mocha --require @babel/register`

Reply:
I did, but the issue was that the default babel preset was for the es5 modules support. I've updated it now, having both the esm and cjs packages use a specific babel env, and the tests use the default. That seems to work now.

Reply:
Nice :) yeah all @babel/register does is override the `require` function to compile the files on the fly, so it makes sense that the `modules` version of the files wouldn't be babel-ified because it doesn't use `require`